### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/bigger-applications.md`

### PR DESCRIPTION
**Pull request by marcelomarkus**
- Portuguese translation:
docs/pt/docs/tutorial/bigger-applications.md

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/tutorial/bigger-applications.md [hunk 1]
```diff
@@ -0,0 +1,556 @@
+# Aplicações Maiores - Múltiplos Arquivos
+
+Se você está construindo uma aplicação ou uma API web, é raro que você possa colocar tudo em um único arquivo.
+
+**FastAPI** oferece uma ferramenta conveniente para estruturar sua aplicação, mantendo toda a flexibilidade.
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/bigger-applications.md:7]: I think it's nice to keep the translations of tip too:

`/// info | "Informação"`

Other translations are doing this:

example

File: docs/pt/docs/tutorial/bigger-applications.md [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+│       └── admin.py     # "admin" submódulo, e.g. import app.internal.admin
+```
+
+## `APIRouter`
+
+Vamos supor que o arquivo dedicado a lidar apenas com usuários seja o submódulo em `/app/routers/users.py`.
+
+Você quer manter as *operações de rota* relacionadas aos seus usuários separadas do restante do código, para mantê-lo organizado.
+
+Mas ele ainda faz parte da mesma **FastAPI** aplicação/web API (faz parte do mesmo "pacote Python").
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/bigger-applications.md:80]: I think it makes more sense to put FastAPI after *aplicação/web API*:

`Mas ele ainda faz parte da mesma aplicação/web API **FastAPI** `

File: docs/pt/docs/tutorial/bigger-applications.md [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+
+E então fazemos o mesmo para o módulo `users`.
+
+Também poderíamos importá-los como:
+
+```Python
+from app.routers import items, users
+```
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/bigger-applications.md:384]: The same here:

`/// info | "Informação"`

File: docs/pt/docs/tutorial/bigger-applications.md [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+
+### Incluir o `APIRouter`s para `usuários` e `itens`
+
+Agora, vamos incluir os `roteadores` dos submódulos `usuários` e `itens`:
+
+```Python hl_lines="10-11" title="app/main.py"
+{!../../../docs_src/bigger_applications/app/main.py!}
+```
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/bigger-applications.md:431]: The same:

`/// info | "Informação"`

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great! ✨

**Review by nazarepiedady — APPROVED** [review]

**Review by ceb10n — APPROVED** [review]

**Comment by alejsdev** [comment]
Nice! Thanks for your contribution @marcelomarkus :nerd_face: 
And thanks for the reviews @ceb10n @nazarepiedady :v:

**Pull request merged**